scripts/multisig_tx.py

```diff
@@ -0,0 +1,338 @@
+"""
+Usage:
+    brownie run multisig_tx build input.csv [prod!]
+    brownie run multisig_tx check 0xsafeTxHash input.csv
+"""
+import csv
+import os
+from hashlib import sha256
+from typing import NamedTuple, Sequence, TypedDict
+
+from ape_safe import ApeSafe, SafeTx
+from brownie import ERC20  # type: ignore
+from brownie import VestingEscrow  # type: ignore
+from brownie import VestingEscrowFactory  # type: ignore
+from brownie import VotingAdapter  # type: ignore
+from brownie import Contract, chain, network
+from brownie.network.transaction import TransactionReceipt
+from gnosis.safe.signatures import signature_split, signature_to_bytes
+from web3._utils.encoding import to_json
+
+from utils import log
+from utils.helpers import chain_snapshot
+
+LDO_ADDRESS = "0x5A98FcBEA516Cf06857215779Fd812CA3beF1B32"
+
+
+def build(csv_filename: str, non_empty_for_prod=None):
+    """Build vesting contracts deployment tx by parameters defined in CSV file"""
+    is_prod = bool(non_empty_for_prod)
+    if is_prod:
+        log.warn("SCRIPT RUNNED IN PRODUCTION ENV")
+        if not log.prompt_yes_no("ARE YOU SURE YOU WANT TO CONTINUE?"):
+            log.warn("Script aborted")
+            return
+
+    _assert_mainnet_fork()
+    config = _read_envs()
+
+    log.info("Reading input file")
+    raw_params_list = _read_csv(csv_filename)
+    params_list = tuple(VestingParams.from_tuple(p) for p in raw_params_list)
+
+    safe = ApeSafe(config["SAFE_ADDRESS"])
+
+    factory_address = config["FACTORY_ADDRESS"]
+    factory = VestingEscrowFactory.at(
+        address=factory_address,
+        owner=safe.address,
+    )
+
+    log.info("Checking multisig balance")
+    starting_balance = _ldo_balance(safe.address)
+    vestings_sum = sum(p.amount for p in params_list)
+    assert starting_balance >= vestings_sum, f"Not enough LDOs for vesting, need at least {vestings_sum / 10 ** 18}"
+
+    log.info("Constructing multisend transaction")
```

Comment:
We need to add multisig `approve` TX with the sum of the LDO required for vestings
OR
We can approve infinite LDO once. Pls ask @kadmil

Reply:
Added LDO approve to the sum of vesting under deploy.